### PR TITLE
annotate RBD storageclasses for reclaimspace

### PR DESCRIPTION
**Pull request by iamniting**
Set reclaimspace schedule to weekly for all RBD storageclasses

Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: controllers/storagecluster/storageclasses.go [hunk 1]
```diff
@@ -407,6 +407,7 @@ func newEncryptedCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageC
 	encryptedStorageClassConfig.storageClass.ObjectMeta.Name = generateNameForEncryptedCephBlockPoolSC(initData)
 	// adding a annotation to support smart cloning across namespace for encrypted volume
 	encryptedStorageClassConfig.storageClass.ObjectMeta.Annotations["cdi.kubevirt.io/clone-strategy"] = "copy"
+	encryptedStorageClassConfig.storageClass.ObjectMeta.Annotations["reclaimspace.csiaddons.openshift.io/schedule"] = "@weekly"
```
Comment (Madhu-1) [inline controllers/storagecluster/storageclasses.go:410]: @iamniting can you please add this to all rbd storageclasses ocs operator is creating?

**Review by malayparida2000 — COMMENTED** [review]
If the aim is the cover all the rbd storageclasses we also need to cover the replica-1 storageclass here https://github.com/red-hat-storage/ocs-operator/blob/2939ab079be135530a96933430a1ea946039218c/controllers/storagecluster/storageclasses.go#L343

**Comment by iamniting** [comment]
> If the aim is to cover all the rbd storageclasses we also need to cover the replica-1 storageclass here
> 
> https://github.com/red-hat-storage/ocs-operator/blob/2939ab079be135530a96933430a1ea946039218c/controllers/storagecluster/storageclasses.go#L343

Done

**Review by malayparida2000 — APPROVED** [review]
/lgtm

**Pull request merged**